packages/validator/README.md

```diff
@@ -20,6 +20,9 @@ Where array `messages` contains message objects:
 ```
 If provided OIS is valid, parameter `valid` will be true, however parameter `messages` still might contain messages with `level` set to `warning`. If `valid` is `false`, there will be always one or more error messages. Similarly functions `isApiSpecsValid` and `isEndpointsValid` check only the API and endpoint specifications contained in the OIS.
 
+Validator can be run as a npm script, by providing specifications file that will be validated and template (validator specification structure) file: `npm run validate --specs="[specsFile]" --template="[templateFile]"`
```

Comment:
This wording sounds a bit weird to me so I've tried to reword it a bit. What do you think?

"
The validator can be run as an NPM script, by providing the paths to the JSON file that will be checked and the JSON file to use as the template (validator specification structure). You can run the validator as script using the following command:

```sh
npm run validate --specs="[path-to-specs-json]" --template="[path-to-template-json]"
```
"